R/model.R

```diff
@@ -47,3 +47,44 @@ run_simulation <- function(pop, parameters = NULL, max_age = 100) {
   output
 
 }
+
+#' @title Run the simulation with repetitions
+#'
+#' @param repetitions n times to run the simulation
+#' @param overrides a named list of parameters to use instead of defaults
+#' @param parallel execute runs in parallel
+#' @return data frame for runs
+#' @export
+run_multiple_models <- function(
+  repetitions,
+  overrides = list(),
+  parallel = FALSE
+) {
+
+
+  if (parallel) {
+    fapply <- parallel::mclapply
```

Comment:
Please hold off on implementing parallel running now, it's really hard to get right (for example this approach will not work on windows, running in series - parallel random numbers are another place to get tripped up as they're hard to get right in practice)